css/global.css

```diff
@@ -0,0 +1,10 @@
+.org-organization-admin-pages-entrypoint-card__card,
```

Comment:
Renamed this file to be more semantic once I realized that I would be updating more than just the feed.